Trigger/src/PrescaleEvent_module.cc

```diff
@@ -91,7 +91,7 @@ namespace mu2e
     bool retval(false);
     bool condition = e.event() % ps == 0;
```

Comment:
The remainder operator will produce a NaN if ps is zero. The variable condition will never be true because NaN==0 is always false.  Please add some code to the c'tor that checks for legal values of the prescal and takes appropriate action.   

Also, I had to look up the order of operator precedence to be sure this was doing the right thing.  I would add () as: ( e.event() % ps) == 0;  your call.

Reply:
two lines below there is a check on `ps` (ps>0), which I think is enough. We will using `ps=-1` as a convention in the Trigger Menu configuration to indicate a trigger path that is non active (in a given event-mode).